README.md

```diff
@@ -1,32 +1,28 @@
 # Hadouken
-> An AWS SQS message processor for Java. Inspired by [Shoryuken] Ruby gem.
+> An RxJava-based AWS SQS message observation library. Inspired by [Shoryuken] Ruby gem.
 
 ![hadouken][img]
 
-![master][master-status]
-![dev][dev-status]
-
 Hadouken for Java combines [Reactive Extensions for Java][rx] with the [AWS SDK for Java][aws-sdk] to create an observable sequence of SQS messages. Resulting application code is short and to the point.
 
-- Transform message contents before the handler fires (optional; comes with support for SNS; can be any `String` &rarr; `String` lambda)
+- Transform message contents before the handler fires (comes with support for SNS, but other transformers are possible by extending `ClientFacade`)
 - Handle transform and/or handler errors with custom logic
-- Automatically acknowledge messages after handler finishes successfully (can be disabled: `options.setAutoAcknowledge(false)`)
 
 ### Getting Started
 
 - [Configure your environment for AWS][aws-config]
-- Add a dependency to this library: `compile 'com.ica-carealign:hadouken:0.0.0'`
+- Add a dependency to this library: `compile 'com.icacarealign:hadouken:x.y.z'`
```

Comment:
You're right, I need to fix that, as well as the example in `README.md`.  I'll do that shortly.